src/Venflow/Venflow/Database.cs

```diff
@@ -63,8 +60,8 @@ protected Database(DatabaseOptionsBuilder optionsBuilder)
         {
             ConnectionString = optionsBuilder.ConnectionString;
             DefaultLoggingBehavior = optionsBuilder.DefaultLoggingBehavior;
-            _loggers = optionsBuilder.Loggers;
-
+            _loggers = optionsBuilder.Loggers.Count > 0 ? optionsBuilder.Loggers : Array.Empty<LoggerCallback>();
```

Comment:
We no longer need to give it a default value, since it is being set here: https://github.com/TwentyFourMinutes/Venflow/blob/9d315fc06c7f1e04f5fd90b6dceebfe0149c6158/src/Venflow/DatabaseOptionsBuilder.cs#L96